internal/http/services/appprovider/appprovider.go

```diff
@@ -153,6 +153,13 @@ func (s *svc) handleNew(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// TODO(lopresti) if target is relative, currently the gateway fails to identify a storage provider (?)
```

Comment:
We do support relative references but I don’t know if web can work with those

For now, it’s good enough